frontends/web/src/routes/account/walletconnect/dashboard.tsx

```diff
@@ -106,14 +106,16 @@ export const DashboardWalletConnect = ({ code, accounts }: TProps) => {
               {hasSession &&
             <div className={styles.sessionCardsContainer}>
               <p className={styles.allSessionsHeading}>{t('walletConnect.dashboard.allSessions')}</p>
-              {sessions.map(session =>
-                <WCSessionCard
-                  key={session.topic}
-                  receiveAddress={getAddressFromEIPString(session.namespaces['eip155'].accounts[0])}
-                  metadata={session.peer.metadata}
-                  onDisconnect={() => handleDisconnectSession(session.topic)}
-                />
-              )}
+              {sessions.map(session => {
+                return (
+                  <WCSessionCard
+                    key={session.topic}
+                    receiveAddress={session.namespaces['eip155'].accounts[0] ? getAddressFromEIPString(session.namespaces['eip155'].accounts[0]) : ''}
```

Comment:
in this case the receiveAddress is an emtpy string, I guess that is intentional… but I don't really understand how this dapp operates. Do they already have the account / address or do they not need it, or will they add it to the session later when needed?

other than this question it LGTM

Reply:
Yes, they already have all the info from the wallet connect pairing, they just don't construct the session namespaces with all the account details